simtools/simtel/simtel_runner_array.py

```diff
@@ -168,6 +182,30 @@ def getOutputFile(self, run):
         )
         return self._simtelDataDir.joinpath(fileName)
 
+    def hasSubLogFile(self, run, mode='out'):
+        runSubFile = self.getSubLogFile(run=run, mode=mode)
+        return Path(runSubFile).is_file()
+
+    def getResources(self, run):
```

Comment:
This method looks like it's the same as the one before where I made a comment on the efficiency of reading, except in this case we are reading in reversed order which I assume would be much more efficient. Perhaps reading in reverse order is faster than my suggestion above.

Either way, can these two functions be made into one general one instead?

Also, docstring =)